components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -610,9 +610,9 @@ public static String getUserConsentURL(OAuth2Parameters params, String loggedInU
             }
 
             if (isOIDC) {
-                consentPage = OAuth2Util.OAuthURL.getOIDCConsentPageUrl();
+                consentPage = IdentityUtil.resolveURL(OAuth2Util.OAuthURL.getOIDCConsentPageUrl(), true, true);
```

Comment:
Should we do this?
With changes done by @ChamathNS, OAuth2Util.OAuthURL.getOIDCConsentPageUrl() should return the tenant qualified URLs, isn't it?

Reply:
This should be fixed in OAuth2Util.OAuthURL such that URL is resolved always an tenant url supported mode, until @ChamathNS get that fixed from the API